kodata/certificates.yaml

```diff
@@ -15,7 +15,7 @@ metadata:
 spec:
   dnsNames:
   - shipwright-build-webhook.shipwright-build.svc
-  - shipwright-build-webhook.shipwright-build.svc.cluster.local
+  - shp-build-webhook.shipwright-build.svc
```

Comment:
Can you explain this ? The [service](https://github.com/shipwright-io/build/blob/main/deploy/402-service-webhook.yaml) is named "shp-build-webhook". Should not there only that be in? Also, are you sure that we never need cluster.local ? I never questioned it, but we basically always create certificates for our webhooks with svcname, svcname.namespace, svcname.namespace.svc, and svcname.namespace.svc.cluster.local like in https://github.com/shipwright-io/build/blob/main/hack/setup-webhook-cert.sh#L32-L35.

Reply:
fixed